src/tests/multihost/alltests/test_services.py

```diff
@@ -20,6 +20,10 @@ def exceute_cmd(multihost, command):
     return cmd
 
 
+def have_files_provider(multihost):
+    cmd = multihost.master[0].run_command('man sssd-files | grep files', raiseonerr=False)
+    return cmd.returncode == 0
```

Comment:
I think it's better to take uniform approach and repeat what is done in
https://github.com/SSSD/sssd-test-framework/blob/master/sssd_test_framework/hosts/client.py#L44
for feature detection.

Reply:
The same question everywhere.

Reply:
Madhuri is taking the uniform approach here:
https://github.com/SSSD/sssd/blob/master/src/tests/multihost/basic/test_files.py#L12
It might be worth to refactor and move the code somewhere else like to helper functions to have it in one place.

Reply:
I added this, MR,
https://github.com/SSSD/sssd/pull/7184